app/controllers/concerns/api/v1/error.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    module Error
```

Comment:
@hoangmirs Lets change name to something more meaningfull, like `ErrorHandler`, or `ErrorMapper`

Reply:
@ankitkalia1195 Fixed in https://github.com/nimblehq/nimble-survey-web/pull/25/commits/4cb2a4a958a5273c2e6148242ccca39bc4844443